tools/setup_helpers/nvtoolext.py

```diff
@@ -0,0 +1,24 @@
+import os
+import platform
+import ctypes.util
+from subprocess import Popen, PIPE
+
+from .env import check_env_flag
+
+WINDOWS_HOME = 'C:/Program Files/NVIDIA Corporation/NvToolsExt'
```

Comment:
Same as CUDA home. Does it change across versions?

Reply:
No, this one is constant generally. But it may change since user can specify this one

Reply:
This one can be queried through registry. It's stored in somewhere like `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer\UserData\S-1-5-18\Components\2221AD5AF46D01746B10434547D0B4F7`.